shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -459,4 +498,26 @@ object JSBackend {
     case DecLit(value) => s"DecLit($value)"
     case StrLit(value) => s"StrLit($value)"
   }
+
+  /** Make a function for `with` construct.
+    *
+    * @param name
```

Comment:
Remove this, it's completely useless :^)

Reply:
Okay. Instead I will put the generated code in this comment.